admin/services/data-access/pupil-census-import.data.service.js

```diff
@@ -22,7 +22,7 @@ pupilCensusImportDataService.sqlBulkImport = async (pupilData, schools, jobId) =
   try {
     result.output = await bulkLoadData(con, pupilData, schools, jobId)
   } catch (error) {
-    result.errorOutput = error
+    result.errorOutput = error && error.message
```

Comment:
if this works, then great.  but i think you could make this more robust because if error is undefined you will have no idea what has happened.  i would create a function, add some object checks to it, try and parse it etc and definitely return something useful if the error object is empty or null to say 'object undefined' or 'object has no error information'

Reply:
@GuyHarwood this has been updated based on your feedback.

Reply:
@GeorgeChatzigiannis could we not be more explicit and call toString on the object if its defined?  This seems a little obtuse to me

Reply:
@GuyHarwood certainly, it's updated.